qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -217,19 +217,20 @@ def shutdown(self):
         for tab in self.widgets():
             self._remove_tab(tab)
 
-    def close_tab(self, tab):
+    def close_tab(self, tab, add_undo=True):
```

Comment:
Please make this a keyword-only argument by adding a `*` argument before `add_undo`.